Digital Twin/procedures/cb.getFloorById.vail

```diff
@@ -0,0 +1,3 @@
+PROCEDURE cb.getFloorById(id String)
```

Comment:
All of these getFloor related procedures should be organized into a service like `Building`. The `cb` service name is a little too generic